Update readme.md

benodonohue opened the pull request:
Hopefully this is right this time..

benodonohue committed:
Update readme.md

Hopefully this is right this time..

jescalan commented on readme.md:
```diff
@@ -8,22 +8,39 @@ An alternate template for [spike](https://github.com/static-dev/spike) that uses
+	? What is the name of your project? 
+	? Describe your project 
+	? What is your github username? 
+	? Would you like a production config file? (Y/n)
```
So I still don't see the need to repeat this exact same thing twice. Perhaps we can just put it there once closer to the top?

benodonohue commented:
Yeah that sounds good.
On Fri, 20 Jan 2017 at 2:44 am, Jeff Escalante <notifications@github.com>
wrote:

> *@jescalan* commented on this pull request.
> ------------------------------
>
> In readme.md
> <https://github.com/static-dev/spike-tpl-roots/pull/4#pullrequestreview-17484302>
> :
>
> >
>  [Spike](https://github.com/static-dev/spike) uses [sprout](https://github.com/carrot/sprout) internally to generate it's project templates. This means you can even use this template without [spike](https://github.com/static-dev/spike) by using [sprout](https://github.com/carrot/sprout) directly.
>
>  - `npm i sprout-cli -g`
> -- `sprout add spike-tpl-roots git@github.com:static-dev/spike-tpl-roots.git`
> -- `sprout new spike-tpl-roots <myproject>`
> +- `sprout add spike-tpl-roots https://github.com/static-dev/spike-tpl-roots.git` <https://github.com/static-dev/spike-tpl-roots.git>
> +- `cd <projectpath>`
> +- `sprout new spike-tpl-roots <projectname>`
> +
> +##### Options
> +
> +	? What is the name of your project?
> +	? Describe your project
> +	? What is your github username?
> +	? Would you like a production config file? (Y/n)
>
> So I still don't see the need to repeat this exact same thing twice.
> Perhaps we can just put it there once closer to the top?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/static-dev/spike-tpl-roots/pull/4#pullrequestreview-17484302>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHg8XV993Dv6jJzhWX1IXSxqKkBNIoEQks5rT4TagaJpZM4LnumG>
> .
>

benodonohue commented:
I will make the changes when I get home tonight..
On Fri, 20 Jan 2017 at 5:47 am, Ben O'Donohue <benjaminodonohue@gmail.com>
wrote:

> Yeah that sounds good.
> On Fri, 20 Jan 2017 at 2:44 am, Jeff Escalante <notifications@github.com>
> wrote:
>
> *@jescalan* commented on this pull request.
> ------------------------------
>
> In readme.md
> <https://github.com/static-dev/spike-tpl-roots/pull/4#pullrequestreview-17484302>
> :
>
> >
>  [Spike](https://github.com/static-dev/spike) uses [sprout](https://github.com/carrot/sprout) internally to generate it's project templates. This means you can even use this template without [spike](https://github.com/static-dev/spike) by using [sprout](https://github.com/carrot/sprout) directly.
>
>  - `npm i sprout-cli -g`
> -- `sprout add spike-tpl-roots git@github.com:static-dev/spike-tpl-roots.git`
> -- `sprout new spike-tpl-roots <myproject>`
> +- `sprout add spike-tpl-roots https://github.com/static-dev/spike-tpl-roots.git` <https://github.com/static-dev/spike-tpl-roots.git>
> +- `cd <projectpath>`
> +- `sprout new spike-tpl-roots <projectname>`
> +
> +##### Options
> +
> +	? What is the name of your project?
> +	? Describe your project
> +	? What is your github username?
> +	? Would you like a production config file? (Y/n)
>
> So I still don't see the need to repeat this exact same thing twice.
> Perhaps we can just put it there once closer to the top?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/static-dev/spike-tpl-roots/pull/4#pullrequestreview-17484302>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHg8XV993Dv6jJzhWX1IXSxqKkBNIoEQks5rT4TagaJpZM4LnumG>
> .
>
>

benodonohue committed:
Update readme.md

benodonohue committed:
Update readme.md

Maybe best just to remove the options i think, its pretty much self explanatory really.

jescalan commented:
Perfect! Thanks for continually putting in the work to make this happen 🎉

benodonohue commented:
More than happy to contribute mate, roots rocks 🤘Spike is a little
daunting for me at the moment, I just can't get my head around the whole
webpack, cssnext etc thing.
On Mon, 23 Jan 2017 at 12:32 pm, Jeff Escalante <notifications@github.com>
wrote:

> Merged #4 <https://github.com/static-dev/spike-tpl-roots/pull/4>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/static-dev/spike-tpl-roots/pull/4#event-932196325>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHg8XUbGbBh5Dw61bMARi81jJyAMXpNIks5rVAMcgaJpZM4LnumG>
> .
>
